framework/src/play/deps/DependenciesManager.java

```diff
@@ -150,7 +150,7 @@ public void sync(List<File> installed) {
             System.out.println("~");
             System.out.println("~ *****************************************************************************");
             System.out.println(
-                    "~ WARNING: Your lib/ and modules/ directories are not synced with current dependencies (use --sync to automatically delete them)");
+                    "~ WARNING: Your lib/ and modules/ directories are not synced with current dependencies (don't used --nosync to automatically delete them)");
```

Comment:
Remove unnecessary 'd' so should be "(don't use --nosync to automatically delete them)